validator/main.py

```diff
@@ -402,3 +406,67 @@ def validate(
             return FailResult(error_message="No valid topic was found.")
 
         return PassResult()
+
+    def validate_stream(
+        self, value: str, metadata: Optional[Dict[str, Any]] = {}
```

Comment:
This needs to accept `**kwargs` to be compliant with the base `Validator.validate_stream`